test/common_quantization.py

```diff
@@ -451,22 +451,27 @@ def __init__(self, quantized=False):
         if quantized:
             self.mycat = nnq.QFunctional()
             self.myadd = nnq.QFunctional()
+            self.myadd_relu = nnq.QFunctional()
         else:
             self.mycat = nnq.FloatFunctional()
             self.myadd = nnq.FloatFunctional()
+            self.myadd_relu = nnq.FloatFunctional()
             self.mycat.observer = DummyObserver()
             self.myadd.observer = DummyObserver()
+            self.myadd_relu.observer = DummyObserver()
 
     def forward(self, x):
         y = self.mycat.cat([x, x, x])
         z = self.myadd.add(y, y)
-        return z
+        w = self.myadd_relu.add_relu(z, z)
+        return w
 
     @classmethod
     def from_float(cls, mod):
         new_mod = cls(quantized=True)
```

Comment:
why do you need to do this? isn't this done by the flow?

Reply:
Rewrote the tests to match how we test other modules